verify.bash

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+travis_tag=$1
+
+release_version_regex="^[0-9]+\\.[0-9]+\\.[0-9]+$"
+
+# Test: on release, we don't display the github url
+if [[ ${travis_tag} =~ ${release_version_regex} ]]
+then
+  if grep -q '^repo_url:' mkdocs.yml
```

Comment:
I don't think this will work in the way you want it to. I think the workflow you are trying for is make a change, push look using this repo's gh-pages, then tag to release it. I'm not sure what the repo_url does, but if you want it for your review, then you'll have to make changes to release instead of just tagging the exact code you reviewed.